### PR TITLE
[DOC] auto migrate only for default template

### PR DESCRIPTION
**Pull request by stefnestor**
@DanRoscigno I'm not replicating [this](https://www.elastic.co/guide/en/elasticsearch/reference/7.16/migrate-index-allocation-filters.html#on-prem-migrate-to-node-roles) in ECE, is it only for the default template?

### CODE REVIEW

**Comment by DanRoscigno** [comment]
> @DanRoscigno I'm not replicating [this](https://www.elastic.co/guide/en/elasticsearch/reference/7.16/migrate-index-allocation-filters.html#on-prem-migrate-to-node-roles) in ECE, is it only for the default template?

I think you and I tested with modified index templates when we went through this together, didn't we?  I do see that on the Elasticsearch docs they specifically call this out:

[If you use custom index templates, check them after the automatic migration completes and remove any attribute-based allocation filters.](https://www.elastic.co/guide/en/elasticsearch/reference/7.16/migrate-index-allocation-filters.html#cloud-migrate-to-node-roles)

File: docs/reference/data-management/migrate-index-allocation-filters.asciidoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ indices to switch to node roles.
 [[cloud-migrate-to-node-roles]]
 === Automatically migrate to node roles on {ess} or {ece}
 
-If you are using node attributes on {ess} or {ece}, you will be
+If you are using node attributes from the default template {ess} or {ece}, you will be
```
Comment (DanRoscigno) [inline docs/reference/data-management/migrate-index-allocation-filters.asciidoc:27]: @stefnestor , what `template` is this?  Is this an index template from Filebeat, or a deployment template from the orchestration system, or something else?

**Comment by stefnestor** [comment]
re: https://github.com/elastic/elasticsearch/pull/82043#issuecomment-999781483

my bad "default template" referring to ECE's Deployment Template labelled "Default" and not an index template somewhere

File: docs/reference/data-management/migrate-index-allocation-filters.asciidoc (hunk 1 above)
Comment (kilfoyle) [inline docs/reference/data-management/migrate-index-allocation-filters.asciidoc:27]:
```suggestion
If you are using node attributes from the default deployment template in {ess} or {ece}, you will be
```

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚀 
I pushed one small change for the text to state explicitly that it's a `deployment template`.

**Pull request merged**